apps/web/app/(authenticated)/(app)/app/settings/root-keys/[keyId]/permissions/permissions.ts

```diff
@@ -8,23 +8,43 @@ type UnkeyPermissions = {
 };
 
 export const workspacePermissions = {
```

Comment:
split this into 2 sections just like the per-api card and also added the missing permissions

![CleanShot 2024-01-29 at 10 36 41@2x](https://github.com/unkeyed/unkey/assets/18246773/247438e2-e6de-4a3b-8ddc-58b57a8f8551)